### ISSUE TITLE
Notes dropped out when exporting as .mid

### ISSUE THREAD

**Issue by FalCT60** [muse-sequencer/muse#1048]
Hello,
As reported in [this thread](https://linuxmusicians.com/viewtopic.php?p=136822#p136822), exporting as .mid for the specific song in attachment makes notes to be dropped-out on track 1.
Well, it seems I cannot attach the archive containing the files, but it is available in the first post of the thread. Sorry.
Rgrds,

**Comment by terminator356** [comment]
Hello. Sorry for the very late reply.
I examined the supplied original and exported midi files.
I see that notes are missing from track 1 **and** track 2.
Therefore I presume notes are missing from **all** tracks.

Using the latest code, I exported the original here and found no trouble. Identical.
Are you still having trouble with this?

**Comment by FalCT60** [comment]
Hello Tim,
Let me first wish a very happy new year to you, together with the whole community too.
Do not worry about the late reply, I perfectly understand that everyone does his best with his means.
OK, I wanted to check the last version that I compiled, but it has disappeared from my computer - if there were not still the structure, I could have thought I dreamt it !
Compile finished some minutes ago, code from git master | 4.0.0-156-gcc133ff0, but still facing the notes drop.
I really do not understand, as it should be fixed. :-(
Well, I just noticed that it does not only occur while exporting to midi, but even on saved projects !
I listened to a work I did on this same file some times ago (did some cuts and pastes), and the issue is there.
I intend to restart from scratch with the file, to ensure I did not do something wrong at some point, but I feel really perplexed.
I am wondering whether it could not be related to the unusual signature of the song, as it does not seem to happen with... OK, tested with another song : same happens, notes are dropped at export.
Really strange. :-/
What else could I try or provide ?

**Comment by terminator356** [comment]
Belated happy new year.

Time signature should not be a problem for import or export or playback.

Let's check a few things.
Export timing is governed by Settings > Midi File Import/Export > Export tab > Division.
By default it is 384. Is it set to some other value?
All check-boxes are usually checked in that tab, except "Use 2-byte time signatures".

Playback timing is derived from the midi division setting
 in Settings > Global Settings > Midi tab > Midi Resolution, which by default is 384.
Is it set to some other value?
There is also a Real Time Clock (RTC) setting there, default is 1024.

When importing, playing, or exporting these midi files, is Jack Audio running?
Or instead, is the song simply using ALSA without Jack Audio (which uses our dummy audio driver)?

When starting MusE, do you see a warning box saying that midi timing is inadequate (but sufficient)?

When you import and work with these midi files, do you add anything more to the project like
 a Synth Track for playback and an Audio Output track for sound?

I see the tempo of the song is precisely 94.0
Should be OK, but (just a theory - if it is a bug) maybe 94.0 causes slight rounding errors when
 computing note times, causing us to miss them, but this would depend on the Midi Division setting (above).

Interesting, I see a ramped tempo slowdown at bar 139. Should be OK, that is far into the song.

If possible, post your old song that you mentioned didn't work (or some reproduce-able broken section of it) .

Hm, what else... I can't think of anything more to check at this time.
Thanks.

**Comment by FalCT60** [comment]
Hello,
Here is what is set :
- Division : 384 ;
- Only "Use 2-byte..." unchecked ;
- Midi Resolution : 384 ;
- RTC : 4096 (maybe after some testings) ;
- I do not understand well, but just in case : in Settings > Midi Ports/Soft Synts, there is nothing related to JACK, all devices are ALSA type ;
- well, I did not really pay attention to this box, since I do not use audio, and had even discarded it. But now that you mention it, and as it has been reactivated after reinstalling, I took the time and read it thoroughly : "Timing source freq 250hz, recommended 500" and, just beneath : "This could lead to audible timing problems for MIDI". I suddenly feel stupid. :-(
Note, however, that in the very first times I was using MusE, I did not find (not really deeply searched, in fact) how to fix this ;
- I do not add anything, I close the current window and start with an empty one. I do not need any audio part or setting.
Just one thing, however, since you refered to JACK & audio : the reason I did not bother with the alert box about frequency is that I use midi devices to play the songs, so I do not see how it could be affected by the audio settings. Am I wrong ?
